x-pack/test/functional/apps/ml/anomaly_detection_jobs/job_list.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Job } from '@kbn/ml-plugin/common/types/anomaly_detection_jobs';
+import { JOB_STATE } from '@kbn/ml-plugin/common';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { getCommonRequestHeader } from '../../../services/ml/common_api';
+import { USER } from '../../../services/ml/security_common';
+
+export default function ({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+  const supertest = getService('supertestWithoutAuth');
+
+  const jobId = `fq_single_1_job_list_test_${Date.now()}`;
+  const calendarId = `test-calendar-${Date.now()}`;
+  const jobsSummaryId = `jobs_summary_fq_multi_${Date.now()}`;
+
+  const multiMetricJobConfig: Job = {
+    allow_lazy_open: false,
+    model_snapshot_retention_days: 0,
+    results_index_name: '',
+    job_id: jobsSummaryId,
+    description: 'mean(responsetime) partition=airline on farequote dataset with 1h bucket span',
+    groups: ['farequote', 'automated', 'multi-metric'],
+    analysis_config: {
+      bucket_span: '1h',
+      influencers: ['airline'],
+      detectors: [
+        { function: 'mean', field_name: 'responsetime', partition_field_name: 'airline' },
+      ],
+    },
+    data_description: { time_field: '@timestamp' },
+    analysis_limits: { model_memory_limit: '50mb' },
+    model_plot_config: { enabled: true },
+  };
+
+  describe('job list', function () {
+    this.tags(['ml']);
+
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await ml.testResources.createDataViewIfNeeded('ft_farequote', '@timestamp');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+
+      const JOB_CONFIG = ml.commonConfig.getADFqSingleMetricJobConfig(jobId);
+      const DATAFEED_CONFIG = ml.commonConfig.getADFqDatafeedConfig(jobId);
+
+      await ml.api.createAndRunAnomalyDetectionLookbackJob(JOB_CONFIG, DATAFEED_CONFIG);
+
+      await ml.api.waitForJobState(jobId, JOB_STATE.CLOSED);
+      await ml.api.openAnomalyDetectionJob(jobId);
+
+      const { body, status } = await supertest
+        .post(`/internal/ml/anomaly_detectors/${jobId}/_forecast`)
+        .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+        .set(getCommonRequestHeader('1'))
+        .send({ duration: '1d' });
+      ml.api.assertResponseStatusCode(200, status, body);
```

Comment:
This should be a helper method in the `api` service. There we don't run it in the user context as it's not part of an API integration test but rather used for functional test preparation.